packages/ui/src/locales/ar-eg.json

```diff
@@ -482,6 +483,18 @@
   "student-settings.edit.personal.topics": "قم باختيار المجالات التي تريد التحدث عنها",
   "student-settings.loading": "برجاء الإنتظار... جاري تحميل الإعدادات!",
   "student-settings.error": " عذرًا، حدث خطأ أثناء تحميل  الإعدادات، برجاء المحاولة مرة أخرى",
+  "settings.profile.title": "الإعدادات",
+  "settings.upload.image.label": "قم برفع صورة جديدة",
+  "settings.upload.image.label.mobile": "تغيير الصورة",
+  "settings.upload.image.desc": "يوصى بصيغة JPG أو PNG وGIF بحجم 480 × 480 بكسل على الأقل.",
+  "settings.edit.personal.title": "تعديل الملف الشخصي",
+  "settings.edit.personal.name": "الاسم",
+  "settings.edit.personal.name.placeholder": "احمد محمد اسامه",
+  "settings.edit.personal.email": "البريد الإلكتروني",
+  "settings.edit.personal.email.placeholder": "ahmedmohamed@gmail.com",
+  "settings.edit.personal.phone-number": "رقم الهاتف",
+  "settings.edit.personal.phone-number.placeholder": "قم بكتابة رقم هاتفك",
+  "settings.edit.personal.city": "المدينة",
```

Comment:
I believe these are duplicates. These ids are related to the student settings page. It should be prefixed with `student-sttings` instead of `settings`. Make sure that these ids are no duplicated.